packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -104,8 +104,10 @@ module.exports = {
 		'jsdoc/require-jsdoc': 'off',
 		'jsdoc/require-param-description': 'off',
 		'jsdoc/require-returns': 'off',
+		'jsdoc/require-yields': 'off',
```

Comment:
Since this rule started being validated after the `eslint-plugin-jsdoc` update, I turned it off because there's many cases it was not adding the `@yield`.